python_modules/libraries/dagster-databricks/dagster_databricks/ops.py

```diff
@@ -98,7 +101,7 @@ class DatabricksRunNowOpConfig(Config):
 
     @op(
         ins={"start_after": In(Nothing)},
-        required_resource_keys={"databricks"},
+        required_resource_keys={databricks_resource_key},
```

Comment:
github won't let me comment on lines that aren't modified by this PR, but on line 111, you won't be able to access `context.resources.databricks` in the case that `databricks_resource_key` is not `"databricks"`. Instead, you can follow the pattern seen here: https://sourcegraph.com/github.com/dagster-io/dagster/-/blob/python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py?L282 to access the resource.

Reply:
yeah make sense! Will add those changes as well